src/components/calendar/CalendarPage.jsx

```diff
@@ -0,0 +1,320 @@
+import React, { Component } from 'react';
+import clsx from 'clsx';
+import { withStyles } from '@material-ui/core/styles';
+import EventIcon from '@material-ui/icons/Event';
+import HighlightOffRoundedIcon from '@material-ui/icons/HighlightOffRounded';
+import PeopleIcon from '@material-ui/icons/People';
+import PersonIcon from '@material-ui/icons/Person';
+import {
+  Avatar,
+  Button,
+  ButtonGroup,
+  Chip,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  FormControl,
+  Grid,
+  Input,
+  InputAdornment,
+  InputLabel,
+  Paper,
+  Select,
+  Tab,
+  Tabs,
+  TextField,
+} from '@material-ui/core';
+
+const styles = theme => ({
+  title: {
+    fontDuration: 'normal',
+    marginLeft: '30px',
+  },
+  flex: {
+    display: 'flex',
+    [theme.breakpoints.down('sm')]: {
+      display: 'block',
+    },
+  },
+  calendar: {
+    // margin: '0 10px',
+    height: '700px',
+  },
+  component: {
+    width: 'calc(100% / 7)',
+    color: '#fff',
+    height: 'fit-content',
+    textAlign: 'center',
+    [theme.breakpoints.down('sm')]: {
+      width: 'inherit',
+    },
+  },
+  label: {
+    padding: '7.5px 0',
+    backgroundColor: '#280e3a',
+    margin: '0',
+  },
+  bar: {
+    width: '5px',
+    height: '84px',
+    backgroundColor: '#e91e63',
+    borderRadius: '15px',
+  },
+  detail: {
+    color: '#000',
+    fontSize: '12px',
+    textAlign: 'left',
+    lineHeight: '1',
+    marginLeft: '10px',
+  },
+  event: {
+    display: 'flex',
+  },
+  header: {
+    justifyContent: 'space-between',
+  },
+  button: {
+    border: 'none',
+    cursor: 'pointer',
+    display: 'flex',
+    fontSize: '16px',
+    outline: 'none',
+    [theme.breakpoints.down('sm')]: {
+      margin: 'auto',
+    },
+  },
+  icon: {
+    margin: 'auto',
+  },
+  iconLabel: {
+    marginLeft: '10px',
+    marginRight: '30px',
+  },
+  duration: {
+    padding: '5px 30px',
+    borderColor: '#765ea8',
+  }
+});
+
+class CalendarPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+      background: ['#280e3a', '#fff', '#fff', '#fff'],
+      color: ['#fff', '#000', '#000', '#000']
+    };
+  }
+
+  handleOpen() {
+    this.setState({ open: true });
+  }
+
+  handleClose() {
+    this.setState({ open: false });
+  }
+
+  handleDelete() {
+
+  }
+
+  handleChange() {
+
+  }
+
+  handleSelect(selected) {
+    let background = this.state.background;
```

Comment:
These two lines of code can be combined into one by doing const { background, color } = this.state;